xbmc/video/test/TestStacks.cpp

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright (C) 2022 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "FileItem.h"
+#include "filesystem/Directory.h"
+#include "test/TestUtils.h"
+
+#include <string>
+
+#include <gtest/gtest.h>
+
+using namespace XFILE;
+
+namespace
+{
+const std::string VIDEO_EXTENSIONS = ".mpg|.mpeg|.mp4|.mkv|.mk3d|.iso";
+}
+
+class TestStacks : public ::testing::Test
+{
+protected:
+  TestStacks() = default;
```

Comment:
not sure if really needed. e.g. TestDateTime also defines the destructor
https://github.com/xbmc/xbmc/blob/9bcb8c6a31da1189958daebeedf12dfddeee80d5/xbmc/test/TestDateTime.cpp#L18-L23